test/cctest/node_test_fixture.h

```diff
@@ -88,6 +90,8 @@ class NodeTestFixture : public ::testing::Test {
   }
 
   virtual void TearDown() {
+    isolate_->Dispose();
```

Comment:
Yes, it looks like it is currently being leaked. I think dispose is needed or at least I don't think it hurts to have it. 

I'm not familiar with Coverity. Is there a way to get this at this information so I can try to clean up any future mess I create? I've looked at https://scan.coverity.com/projects/node-js, but can't really get to much information, I've requested to be added to the project so perhaps once that is approved I might be able to see some more details. 

Sorry about this and thanks for fixing it.